comptests/s3/test_rest_api_rgw.py

```diff
@@ -58,18 +112,171 @@ def test_36622(self):
             "Step 1: Creating a new IAM user with name %s", str(user_name))
         loop = asyncio.get_event_loop()
         status, user_info = loop.run_until_complete(self.obj.create_user(user_params))
-        self.log.info(user_info)
+        assert status == HTTPStatus.OK , "Not able to create user. Test Failed"
+        self.log.info("Created user details: %s",user_info)
+        self.log.info(
+            "Step 2: Verifying that new IAM user is created successfully")
+        status, user_info = loop.run_until_complete(self.obj.get_user_info(user_params))
+        assert status == HTTPStatus.OK , "Not able to Get user Info. Test Failed"
+        self.log.info("Get user info output: %s",user_info)
+        self.created_users.append(user_params)
+        self.log.info("END: Test create iam user specifying uid and display name and email. : %s",test_case_name)
+
+    @pytest.mark.api_user_ops
+    @pytest.mark.tags('TEST-36632')
+    def test_user_create_36632(self):
+        """Create new IAM user."""
+        test_case_name = cortxlogging.get_frame()
+        self.log.info("##### Test started -  %s #####", test_case_name)
+        self.log.info("START: Test create iam user specifying just uid .")
+        user_name = f"{self.user_name_prefix}{str(time.perf_counter_ns()).replace('.', '_')}"
+        user_params = {
+            'uid' : user_name
+         }
+        self.log.info(
+            "Step 1: Creating a new IAM user with name %s", str(user_name))
+        loop = asyncio.get_event_loop()
+        status, user_info = loop.run_until_complete(self.obj.create_user(user_params))
+        assert status == HTTPStatus.BAD_REQUEST, "Able to create user with just uid. Test Failed"
+        #self.created_users.append(user_params)
+        self.log.info("END: Test create iam user specifying just uid . : %s",test_case_name)
+
+    @pytest.mark.api_user_ops
+    @pytest.mark.tags('TEST-36633')
+    def test_user_create_36633(self):
+        """Create new IAM user."""
```

Comment:
All your comment says create new IAM user. You may want to add a one liner description of exact scenario.